privateSBOMExchange/src/petra/lib/kms/kms.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, jsonify
+from petra.lib.auth.google_oidc import authenticate_and_get_id_token
+from petra.lib.auth.verify import verify_id_token
+from petra.lib.util.config import Config
+from urllib.parse import urlparse
+import cpabe
+
+app = Flask(__name__)
+
+def cpabe_setup():
+    return cpabe.cpabe_setup()
```

Comment:
This is nitpicky, but is there a reason we need this wrapper function, and can't call `cpabe.cpabe_setup()` directly?